src/features/redirects/redirects.json

```diff
@@ -5,21 +5,6 @@
       "destination": "any-api/api-reference",
       "statusCode": 301
     },
-    {
-      "source": "docs/any-api/data-providers/dns-ownership",
-      "destination": "any-api/data-providers/dns-ownership",
-      "statusCode": 301
-    },
-    {
-      "source": "docs/any-api/data-providers/google-weather",
-      "destination": "any-api/data-providers/google-weather",
-      "statusCode": 301
-    },
-    {
-      "source": "docs/any-api/data-providers/introduction",
-      "destination": "any-api/data-providers/introduction",
-      "statusCode": 301
-    },
```

Comment:
Let's update these redirects to go to the Any API intro. Also, add new redirects for the current URLs. We want to avoid 404s.